docs/installation/integrations/react.md

```diff
@@ -599,6 +599,10 @@ import { createRequire } from 'node:module';
 const require = createRequire( import.meta.url );
 ```
 
+<info-box warning>
```

Comment:
I think this should appear higher, near the: Configuring `vite.config.js` / `vite.config.ts`

Reply:
I'll probably add another section for Vite + custom build, so we don't have to link to comments, etc.

Reply:
Eventually, I added a new section under the Integration a build from the online builder header. Configuring the `vite.config.js` header belongs to the build from the source path, and from my testing, it works fine. The problem is with the custom build from the online builder, so I put the info there.